docs/builder-guides/Use Astar Features/How to use Hardhat to deploy on Shibuya.md

```diff
@@ -0,0 +1,171 @@
+# How to use Hardhat to deploy on Shibuya
+
+## TL;DR
+
+Hardhat is an Ethereum (EVM) development environment that helps developers to manage and automate the recurring tasks inherent to the process of building smart contracts and dApps, which means compiling and testing at the very core. 
+
+Since Astar Network is a multi-VM smart contract hub, we support both WASM and EVM smart contracts, which means you can use Ethereum dev tools including Hardhat to directly interact with Astar EVM’s API and deploy Solidity smart contracts on Astar.
+
+In this cookbook, we will guide you on how to set up the environment for Astar EVM development, how to create and configure a Hardhat project for Astar EVM, and how to deploy a simple Solidity smart contract on Astar EVM via Hardhat.
+
+---
+
+## What is Astar EVM?
+
+As a multi-VM smart contract hub, Astar Network supports both WASM and EVM smart contracts, which means both Solidity smart contracts and WASM-based smart contracts can be deployed on Astar Network. 
+
+And for Solidity developers, you can directly use Ethereum dev tools including Hardhat, Remix, MetaMask to directly interact with Astar EVM’s API and deploy Solidity smart contracts on Astar EVM.
+
+## What is Hardhat?
+
+Hardhat is a development environment that helps developers in testing, compiling, deploying, and debugging smart contracts and dApps on Ethereum Virtual Machine (EVM). Hardhat smart contract development environment offers suitable tools to developers for managing the development workflow, and identifying causes for the failure of applications.
+
+---
+## Set up Node.js environment for Hardhat
+Hardhat is built on top of Node.js, the JavaScript runtime built on Chrome's V8 JavaScript engine. As the first step for to set up Hardhat, we need to set up Node.js environment.
+
+Make sure you have git installed and run the following command in the terminal.
```

Comment:
@SUGURUSEO Which command are you referring here to - it seems the command is missing ?

Reply:
Deleted this sentence.